pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -44,6 +44,7 @@ func (s *Service) SDKToAutoScalingGroup(v *autoscaling.Group) (*expinfrav1.AutoS
 		MaxSize:           int32(aws.Int64Value(v.MaxSize)),
 		MinSize:           int32(aws.Int64Value(v.MinSize)),
 		CapacityRebalance: aws.BoolValue(v.CapacityRebalance),
+		Subnets:           strings.Split(*v.VPCZoneIdentifier, ", "),
```

Comment:
Do we need the split identifier to be `, ` and not `,` ?

Reply:
the VPCZoneIdentifier is made like this `aws.String(strings.Join(i.Subnets, ", "))` hence making the subnets by splitting on `, `  else the subnets length increases by one due to extra space